tcgaImport.py

```diff
@@ -1069,56 +1041,45 @@ class SNP6Import(TCGASegmentImport):
     def fileScan(self, path, dataSubType):
         handle = open(path)
         colName = None
-        for line in handle:
-            if colName is None:
-                colName = line.rstrip().split("\t")                     
-                for i, col in enumerate(colName):
-                    if commonMap.has_key( col ):
-                        colName[i] = commonMap[ col ]
-            else:
-                tmp = line.rstrip().split("\t")
-                out = {}
-                for i in range(1, len(colName)):
-                    out[ colName[i] ] = tmp[i]
-                self.emit( tmp[0], out, dataSubType )
+        line = handle.readline()
+        colName = line.rstrip().split("\t")                     
```

Comment:
pd.read_csv... Skip handling of column names.
Instead use:

tmp.columns = [commonMap.get(col, col) for col in tmp.columns]